docs/source/using-discovery/usingslurm.rst

```diff
@@ -59,42 +103,376 @@ To run a job interactively, use ``srun``. To submit a job to run in the backgrou
    * - ``scancel <jobid>``
      - Cancel a pending or running job on the cluster
 
-Monitoring Jobs
+.. _using_srun:
+Using srun
+-----------
+Use the Slum command ``srun`` to allocate an interactive job. This means you use specific options with ``srun`` on the command line to tell Slurm what resources are needed to run your job, such as number of nodes, amount of memory, and amount of time. Enter ``srun`` command and options on the command line, the and press ``Return``. Slurm will find, and then allocate, the specified resources. Depending on the specifications, this process may take a few minutes. All ``srun`` options can be found in the `Slurm documentation`_. The following image shows an example of an ``srun`` command as run on a command line.
+
+.. image:: /images/srun_example.jpg
+  :alt: image of the command line showing an example srun command
+
+Example uses
```

Comment:
Can we change these "Example uses" headings to be more specific, like srun examples?